source/From/JavaScript.mint

```diff
@@ -47,7 +47,13 @@ module From {
                 FromItem.String("var x = 42\nlet x = 42\nconst x = 42"),
               "Constants" => FromItem.String("const THE_ANSWER = 42"),
               "String concatenation" => FromItem.String(
-                "\"Hello\" + \"World!\""),
+                  <<~JAVASCRIPT
+                  "Hello" + "World!"
+
+                  let hello = "Hello"
+                  let world = "World"
+                  `${hello} ${world}!`
+                  JAVASCRIPT),
```

Comment:
The feature is only about concatenation, this is interpolation which can be another row in the comparison.